tests/transforms/test_mitigate.py

```diff
@@ -154,6 +161,9 @@ def ideal_circuit(w1, w2):
         res_mitigated = qml.math.stack(res_mitigated)
         res_ideal = qml.math.stack(res_ideal)
 
+        print(res_ideal)
+        print(res_mitigated)
```

Comment:
```suggestion
```

Reply:
Sorry about leaving these in! Removed here, and the other print calls in https://github.com/PennyLaneAI/pennylane/pull/5972/commits/58d1415a5ed0e2e3c5a38f0dd1a8818d532faa16.